src/main/kotlin/org/rust/cargo/toolchain/RustToolchain.kt

```diff
@@ -108,31 +100,16 @@ data class RustToolchain(val location: String) {
     }
 }
 
+open class Version(val major: Int, val minor: Int, val patch: Int) {
+    override fun toString(): String = "$major.$minor.$patch"
+}
```

Comment:
Thanks! I've completely forgotten that I wanted to use it the next time I refactor this!